ta/mk/ta_dev_kit.mk

```diff
@@ -9,7 +9,19 @@ include $(ta-dev-kit-dir)/mk/conf.mk
 
 binary := $(BINARY)
 
-CROSS_COMPILE_$(sm)	?= $(CROSS_COMPILE)
+# Convenience/compatibility
+CROSS_COMPILE32		?= $(CROSS_COMPILE)
+CROSS_COMPILE64		?= $(CROSS_COMPILE)
+
+ifeq ($(sm),ta_arm32)
```

Comment:
So far we've been able to avoid arch dependent stuff in this file, perhaps this information can be passed in the generated export-$(sm)/mk/conf.mk